BUILD.bazel

```diff
@@ -61,3 +62,15 @@ buildifier(
     lint_mode = "warn",
     mode = "diff",
 )
+
+bool_flag(
+    name = "flag_bzlmod",
```

Comment:
comment about why we need this strange thing

Reply:
and, we should file the issue if someone hasn't already, that it should be possible to sense this like we do for `--stamp` https://github.com/aspect-build/bazel-lib/blob/main/lib/private/stamping.bzl#L29

Reply:
@Wyverald didn't think that could be easily done.

Reply:
Added some clarifying comments.